src/excmds.ts

```diff
@@ -4023,15 +4034,48 @@ export function autocontain(...args: string[]) {
     if (urlMode || saneMode) {
         args.splice(0, 1)
     }
-    if (args.length !== 2) throw new Error("syntax: autocontain [-{u,s}] pattern container")
+    if (args.length < 2) throw new Error("syntax: autocontain [-{u,s}] pattern container proxy1 proxy2")
 
-    let [pattern, container] = args
+    let [pattern, container, ...proxies] = args
 
     if (!urlMode) {
         pattern = saneMode ? `^https?://([^/]*\\.|)${pattern}/` : `^https?://[^/]*${pattern}/`
     }
 
-    return config.set("autocontain", pattern, container)
+    proxies.length && Proxy.exists(proxies)
+
+    return config.set("autocontain", pattern, proxies.length ? [container, proxies.join(",")].join("+") : container)
+}
+
+/** Add a proxy.
+
+ @param name The name of the proxy you want to set
+
+ @param url The proxy URL. List of supported protcols are "http", "https/ssl", "socks/socks5", "socks4".
```

Comment:
I was reading this as literally "https/ssl"

```suggestion
 @param url The proxy URL. List of supported protcols are "http", "https" or equivalently "ssl", "socks5" or equivalently "sock" and "socks4".
```